### DISCUSSION TITLE
Govee Update for Stream Deck

### DISCUSSION THREAD

**Discussion by SSGTGames** [community/community#88037] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Good day. 
     I am reaching out as I love your current Stream Deck application for Govee but was curious as to if it is possible to add an integration for their H6602 Govee AI Gaming Sync Box.  I would love to be able to change inputs as well as the current functions.
     
     Thank you.

**Comment by queenofcorgis** [comment]
Hi @SSGTGames ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Govee and their StreamDeck applicatoin. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
